admin/backend/routes.py

```
@@ -0,0 +1,988 @@
+            "message": f"File '{filename}' updated successfully",
+            "filename": filename,
+            "size": file_path.stat().st_size,
+            "modified": datetime.fromtimestamp(file_path.stat().st_mtime).isoformat(),
+        }
+    except Exception as e:
+        # Restore from backup if write failed
+        try:
+            if backup_path.exists():
+                shutil.copy2(backup_path, file_path)
+                backup_path.unlink()
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)